.github/workflows/publish.yml

```diff
@@ -0,0 +1,15 @@
+name: Publish
+
+on:
+  push:
+    tags: ["*"] # Releases
```

Comment:
One more thing: We should also add `branches:...` like [here](https://github.com/playframework/playframework/blob/becb8ccb40f3adb5bd1b285ec70fb62cf17a5768/.github/workflows/publish.yml#L5-L6).
This way snapshots get published each time a pull request gets merged. That's quite nice to have. (akka does the same btw)